helm/values-uat.yaml

```diff
@@ -28,7 +28,7 @@ microservice-chart:
         TRANSACTIONS_AUTHORIZATION_REQUESTED_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-auth-requested-queue-b
         TRANSACTIONS_AUTHORIZATION_OUTCOME_WAITING_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-auth-outcome-waiting-queue-b
         NODO_ECOMMERCE_CLIENT_ID: ecomm-blue
-        OTEL_RESOURCE_ATTRIBUTES: "service.name=pagopa-ecommerce-event-dispatcher-service-blue,service.deployment.environment=uat"
+        OTEL_RESOURCE_ATTRIBUTES: "service.name=pagopa-ecommerce-event-dispatcher-service-blue,deployment.environment=uat"
```

Comment:
watch out: change also the value at line 144. this comment applies also to prod value, change all occurrences